sdk/js/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@apeworx/apepay",
+  "version": "0.1.0-alpha.0",
+  "author": "ApeWorX LTD.",
+  "license": "Apache-2.0",
+  "type": "module",
+  "devDependencies": {
+    "typescript": "^5.1.6"
```

Comment:
So, using workspaces I think the top level package.json's `dependencies` are actually dev dependencies for the entire project (keeping them aligned), and then down here this would be prod dependencies for the package itself

That being said, I think this should be removed because typescript is only a dev dependency